src/main/java/freedom/tasks/Event.java

```diff
@@ -0,0 +1,62 @@
+package freedom.tasks;
+
+import freedom.exceptions.DescriptionEmpty;
+import freedom.exceptions.TimeEmpty;
+
+public class Event extends Task{
+    protected String from;
+    protected String to;
```

Comment:
Perhaps using more specific names such as startTime and endTime might be better here.